pkg/controller/controlplane/valuesprovider_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("ValuesProvider", func() {
 				Raw: encode(&apisawsv1alpha1.ControlPlaneConfig{
 					CloudControllerManager: &apisawsv1alpha1.CloudControllerManagerConfig{
 						FeatureGates: map[string]bool{
-							"CustomResourceValidation": true,
+							"RotateKubeletServerCertificate": true,
```

Comment:
I am not sure, is this a valid CCM flag ? Because I see this was changed in many places.

Reply:
I just saw this in `charts/internal/seed-controlplane/charts/cloud-controller-manager/values.yaml` together with  `CustomResourceValidation `, so I just replaced it in examples and tests.